src/styles/variables.scss

```diff
@@ -27,10 +27,10 @@ $white: white;
 $green: #008000;
 $jungle-green: #29ab87;
 $active-green: #87d870;
-$yellow: #ecef08;
+$yellow: #eab308;
 $orange: #ffa500;
 $light-red-1: #ffb0b0;
-$light-red-2: #ffdddd;
+$light-red-2: #fef2f2;
```

Comment:
Same as above

Reply:
what is same as above? have deleted the hex and added a new.

Reply:
Might affect other than task card items?
If not we can ignore it

Reply:
Please try not adding comments like `Same as above`, it confuses the `Author`. If you want to reference add comments beforehand then copy the link of that comments which are repeated and add comment links in the next ones

Reply:
Also be specific to what changes are you referring to also helps